manager/integration/tests/test_backing_image.py

```diff
@@ -421,3 +421,26 @@ def test_exporting_backing_image_from_volume():  # NOQA
         is the same as that of the 2nd volume.
     11. Do cleanup.
     """
+
+
+@pytest.mark.skip(reason="TODO") # NOQA
+@pytest.mark.backing_image  # NOQA
+def test_csi_snapshot_support(client, volume_name):  # NOQA
```

Comment:
Instead of having one test case with all the scenarios of `bak`, `snap` and `invalid`. Can we have 3 test cases for each case? e.g. `test_csi_snapshot_with_bak_paraam`

Or if you want to parameterize one test case with `bak`, `snap` and `invalid`, I'm fine with that too.

Reply:
Hi @khushboo-rancher ,

I updated PR for you review, thank you